enterprise/frontend/src/metabase-enterprise/remote_sync/components/ChangesLists/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { t } from "ttag";
+
+import { Box, Loader, Text } from "metabase/ui";
+import { useGetRemoteSyncChangesQuery } from "metabase-enterprise/api";
+import type { Collection } from "metabase-types/api";
+
+import { AllChangesView } from "./AllChangesView";
+
+interface ChangesListsProps {
+  collections: Collection[];
+  title?: string;
+}
+
+export const ChangesLists = ({ collections, title }: ChangesListsProps) => {
```

Comment:
Maybe this is a new convention we're following, but normally our index files just re-export things